code/modules/mob/living/simple_animal/hostile/mining_mobs/lobstrosity.dm

```diff
@@ -0,0 +1,34 @@
+/**
+  *	Lobstrosities, the poster boy of charging AI mobs. Drops crab meat and bones.
+  * Outside of charging, it's intended behavior is that it is generally slow moving, but makes up for that with a knockdown attack to score additional hits.
+  */
+/mob/living/simple_animal/hostile/asteroid/lobstrosity
+	name = "lobstrosity"
+	desc = "A marvel of evolution gone wrong, the frosty ice produces underground lakes where these ill tempered seafood gather. Beware its charge."
```

Comment:
If the lakes are underground, why do these guys only spawn on the top level? Shouldn't you be able to find these lakes on the lower levels (in, say, some form of special ruin with, like, several of these guys in it)? I mean, we have the blood drunk miner's ice lake, but you can't really intentionally put hostile simplemobs in a boss arena (that doesn't have hostile simplemob minions as its gimmick)...

Reply:
Mostly just because I wanted them to be a weaker mob, but not dilute the mob pool of the lower levels. Ya know, mid-level fodder.